### DISCUSSION TITLE
Support for MongoDB !!

### DISCUSSION THREAD

**Discussion by shelcia** [deta/cloud-docs#636] [Polls]
I use mongodb in almost all of my projects. I think it will be wonderful if deta extends its support to it.

**Comment by MMartin09** [comment]
I am using MongoDB with FastAPI and Beanie on Deta. This seems to work fine.

**Reply by abhijithvijayan-kubric** [reply]
where are you hosting mongodb in this case?

**Reply by shelcia** [reply]
Mongo db atlas

**Reply by MMartin09** [reply]
I only tested it on MongDB Atlas. However, you have to note that startup events are not working on Deta. With FastAPI, for example, you thus have to connect to DB manually (e.g., in the endpoints).

**Comment by abdelhai** [comment]
we **don't don't** support mongodb. mongodb atlas is the issue here as they won't allow connections from our micros as they require a static ip.
if you use their http api, i believe you can still talk with your db on atlas.
